api_contracts/contracts/downloadCenter/DownloadCenter.md

```diff
@@ -42,11 +42,81 @@ This endpoint returns data corresponding to the latest data dictionary csv file.
 ### Data Dictonary [GET]
 
 + Response 200 (application/json)
-    + Attributes 
+    + Attributes
         + document (DataDictionary)
-    
+
+
+## Download Count [/api/v2/download/count/]
+
+Returns the number of transactions that would be included in a download request for the given filter set.
+
+### Download Count Data [POST]
++ Request (application/json)
+    + Attributes (object)
+        + filters: (required, object)
+            + time_period (array[TimeRangeObject])
```

Comment:
This endpoint accepts a comprehensive [Filter Object](https://github.com/fedspendingtransparency/usaspending-website/blob/dev/api_contracts/contracts/search/AdvancedSearch.md#filterobject-object), not just `time_period`.